README.md

```diff
@@ -63,7 +63,7 @@ You can now preview the site locally on http://localhost:1313/
 
 To create documentation for a new release of `parquet-format` create a new <releaseNumber>.md file under `content/en/blog/parquet-format`. Please see existing files in that directory as an example.
 
-To create documentation for a new release of `parquet-mr` create a new <releaseNumber>.md file under `content/en/blog/parquet-mr`. Please see existing files in that directory as an example.
+To create documentation for a new release of `parquet-java` create a new <releaseNumber>.md file under `content/en/blog/parquet-java`. Please see existing files in that directory as an example.
```

Comment:
I would personally suggest removing the discussion bout release of `parquet-mr`/ `parquet-java` to that repo. It seems confusing to have instructions on how to do a release from another repo in `parquet-site`

Reply:
Ah I see the confusion - these notes have to do with updating the website announce the new release: https://parquet.apache.org/blog/. So the flow would be
1. Make a release of parquet-java in that repo
2. Put up a blog post entry on the website containing the release information

Reply:
I don't feel strongly